modules/installer/cfg/cfg.go

```diff
@@ -31,22 +31,30 @@ var (
 
 	defaultKubeconfigPath   = fmt.Sprintf("%s/%s", os.Getenv("HOME"), ".kube/config")
 	defaultDebugLogsEnabled = false
-	defaultHelmReleaseName  = "kuberlogic"
-
-	defaultPlatform    = "generic"
-	supportedPlatforms = []string{defaultPlatform, "aws"}
+	defaultPlatform         = "generic"
+	supportedPlatforms      = []string{defaultPlatform, "aws"}
 )
 
 type Config struct {
-	DebugLogs      *bool   `yaml:"debugLogs,omitempty"`
-	KubeconfigPath *string `yaml:"kubeconfigPath,omitempty"`
+	DebugLogs      *bool   `yaml:"debug-logs,omitempty"`
+	KubeconfigPath *string `yaml:"kubeconfig-path,omitempty"`
 
 	Namespace *string `yaml:"namespace"`
 
 	Endpoints struct {
-		API               string `yaml:"api""`
-		UI                string `yaml:"ui"`
-		MonitoringConsole string `yaml:"monitoringConsole"`
+		Kuberlogic    string `yaml:"kuberlogic"`
+		KuberlogicTLS struct {
```

Comment:
What do you think about switching to a *struct definition here?

Later on we can if custom tls is enabled with a simple comparison: `c.Endppoints.KuberlogicTLS != nil`

Reply:
Agree